.travis.yml

```diff
@@ -212,14 +213,13 @@ matrix:
     # Build mdbook documentation
     - name: "doc: Guide documentation"
       install:
-        - mkdir -p $HOME/mdbook
-        - curl -L https://github.com/rust-lang-nursery/mdBook/releases/download/v0.2.1/mdbook-v0.2.1-x86_64-unknown-linux-musl.tar.gz | tar xzf - -C $HOME/mdbook
```

Comment:
In the interest of CI time and a smaller cache, perhaps this could still be downloaded but mdbook-linkcheck is compiled?

Reply:
Do we still save much time/cache if mdbook is a dependency of mdbook-linkcheck? Building mdbook-linkcheck will still build mdbook, but as a library instead of a binary.

Reply:
Oh yeah for sure just caching the binaries is fine, it's caching wasm-bindgen's target directory that's a bit large over time